backend/s3/file.go

```diff
@@ -182,9 +182,12 @@ func (f *File) Delete() error {
 func (f *File) Close() error {
 
 	if f.tempFile != nil {
-		defer func() { _ = f.tempFile.Close() }()
+		err := f.tempFile.Close()
+		if err != nil {
+			return err
+		}
 
-		err := os.Remove(f.tempFile.Name())
+		err = os.Remove(f.tempFile.Name())
 		if err != nil && !os.IsNotExist(err) {
 			return err
 		}
```

Comment:
Note, this was a windows-only issue.  There are test failures in Windows in other backends.  Will add issues for them as well updating CI to include Windows testing.